### PR TITLE
Update repo docs to include pwa-devdocs

### PR DESCRIPTION
**Pull request by jcalcaben**
## This PR is a:

[ ] New feature
[X] Enhancement/Optimization
[ ] Refactor
[X] Bugfix
[ ] Test for existing code
[X] Documentation

## Summary

When this pull request is merged, it will fix links and references to the old repositories in the README and template files. It also adds `pwa-devdocs` related information to the README and template files.

## Additional information

### CODE REVIEW

File: packages/pwa-devdocs/README.md [hunk 1]
```diff
@@ -3,66 +3,71 @@
...
-* [Ruby]
-* [Bundler]
-* [Yarn]
- 
+
+-   [Git]
+-   [NodeJS]
+-   [Ruby]
+-   [Bundler]
+-   [Yarn]
```
Comment (DrewML) [inline packages/pwa-devdocs/README.md]: Did you mean `npm`? I don't see `yarn` usage anywhere here

File: packages/pwa-devdocs/README.md [hunk 2]
```diff
@@ -3,66 +3,71 @@
...
- 
-* [Git]
-* [NodeJS]
-* [Ruby]
-* [Bundler]
-* [Yarn]
- 
+
+-   [Git]
+-   [NodeJS]
```
Comment (DrewML) [inline packages/pwa-devdocs/README.md]: The node project uses `Node.js` rather than `NodeJS`

File: packages/pwa-devdocs/README.md (hunk 1 above)
Comment (brendanfalkowski) [reply packages/pwa-devdocs/README.md]: A bit curious what Ruby is used for.

Comment (DrewML) [reply packages/pwa-devdocs/README.md]: It's used in `pwa-devdocs` to run Jekyll.

Comment (jcalcaben) [reply packages/pwa-devdocs/README.md]: @DrewML 
I can change it to `npm` since the rest of the repo uses it

**Pull request merged**